src/components/States.jsx

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import '../App.css';
+
+const State = ({
+  countryState, icon, weatherCondition, temperature, date,
+}) => (
+  <div className="weather-card">
+    <div className="one">
+      <h3 className="state-name">{countryState}</h3>
+      <h2 className="weather-date">{date}</h2>
+    </div>
+    <div className="2">
+      <h3 className="weather-temp">
+        {temperature}
+        °C
+      </h3>
+    </div>
+    <div className="box-3">
+      <img className="weather-image" src={icon} alt="weatherImage" />
+      <p className="weather-condition">{weatherCondition}</p>
+    </div>
+  </div>
+);
+
+export default State;
+
+State.propTypes = {
+  countryState: PropTypes.string.isRequired,
+  icon: PropTypes.string.isRequired,
+  date: PropTypes.string.isRequired,
+  weatherCondition: PropTypes.string.isRequired,
+  temperature: PropTypes.number.isRequired,
+};
```

Comment:
Good job working on the project requirements. But it would be nice to stick to the template design on behance. Your design should be identical to the template on behance. 

As it say in the requirements

Design


Student should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:

Typography: font face, size, and weight
Layout: composition and space between elements.


Students should select one main colour for the app and be consistent (it does not need to be the same colour as in the design guidelines).


Thiis is your design:

![Screenshot 2023-08-04 at 7 56 16 AM](https://github.com/Alibaba2023/world-weather-app/assets/79506105/050e601f-d1f5-4148-a777-6beac384fc5b)

![Screenshot 2023-08-04 at 7 55 55 AM](https://github.com/Alibaba2023/world-weather-app/assets/79506105/407e4578-3400-4c8c-ad9b-cac49384030a)

It would be nice to stick to this design requirements:

![Screenshot 2023-08-04 at 08-19-43 Ballhead App (Free PSDs)](https://github.com/Alibaba2023/world-weather-app/assets/79506105/c8b82e02-3a4a-4e77-b7fe-6d4aab8b9e40)

Also the requirre font to use for this project is 

Your control font : Lato

Your body and title font should be gill sans 

Please implement the these changes it is important to follow design requirements